helm-chart/metabase/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "metabase.fullname" . }}-test-connection"
+  labels:
+    {{- include "metabase.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args: ['{{ include "metabase.fullname" . }}:{{ .Values.service.externalPort }}']
```

Comment:
This won't work with the current port config (see comment above)

Reply:
solved